modular_zubbers/modules/modular_items/code/idmaco_donator.dm

```diff
@@ -19,26 +19,30 @@
 
 /datum/loadout_item/under/miscellaneous/idmasnowfatigue
 	name = "IDMA service uniform"
-	item_path = /obj/item/clothing/under/idma_fatigue
-	ckeywhitelist = list("especiallystrange", "snaffle15")
+	item_path = /obj/item/clothing/under/rank/security/idma_fatigue
+	ckeywhitelist = list("especiallystrange", "waterpig",  "snaffle15")
+	restricted_roles = list(JOB_BLUESHIELD, JOB_CAPTAIN, JOB_NT_REP, JOB_HEAD_OF_SECURITY, JOB_RESEARCH_DIRECTOR, JOB_CHIEF_MEDICAL_OFFICER, JOB_CHIEF_ENGINEER, JOB_HEAD_OF_PERSONNEL, JOB_QUARTERMASTER, JOB_SECURITY_OFFICER, JOB_WARDEN, JOB_SECURITY_MEDIC) //Secure Personnel Only. It is a round start armor even if the actual armor can be ordered for cheap
```

Comment:
if this is a jumpsuit some minor wound bonus is probably appropriate- all jumpsuits have minor wound defense

Reply:
oh, right, this is the loadout entry... not the actual item- my bad!